### PR TITLE
Add auth blog 

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

File: _posts/2017-05-25-0.1-auth.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+The following diagram explains the end to end Istio Auth authentication workflow on Kubernetes:
+
+<figure>
+    <a href="{{home}}/img/istio_auth_workflow.png">
+    <img style="max-width: 100%;" src="{{home}}/img/istio_auth_workflow.png" alt="" title="Istio Auth Workflow" />
+    </a>
+</figure>
+
+
+Thus, Istio Auth is part of the broader security story for containers. Red Hat, a partner on the development of Kubernetes, has identified [10 Layers](https://www.redhat.com/en/resources/container-security-openshift-cloud-devops-whitepaper) of container security. Istio and Istio Auth addresses two of these layers: "Network Isolation" and "API and Service Endpoint Management". As cluster federation evolves on Kubernetes and other platforms, our intent is for Istio to secure communications across services spanning multiple federated clusters.
```
Comment (sraygoogle) [inline _posts/2017-05-25-0.1-auth.md]: Delete "Thus,"

Comment (wattli) [inline _posts/2017-05-25-0.1-auth.md]: Done

File: _posts/2017-05-25-0.1-auth.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Thus, security administrators need a tool that enables them to consistently, and by default, secure all communication between services across diverse production environments. 
+
+## Solution: Strong service identity and authentication
+
+Google has, over the years, developed architecture and technology to uniformly secure millions of services in its production environment against external attacks and insider threats. Key security principles include trusting the endpoints and not the network, strong mutual authentication based on service identity and service level authorization. Istio Auth is based on the same principles. 
+
+The alpha release of Istio Auth runs on Kubernetes and provides the following features:
```
Comment (geeknoid) [inline _posts/2017-05-25-0.1-auth.md]: Please replace "alpha" with 0.1

File: _posts/2017-05-25-0.1-auth.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+* Management of keys and certificates at scale
+
+Istio Auth is based on industry standards like mutual TLS and X.509. Furthermore, Google is actively contributing to an open, community-driven service security framework called [SPIFFE](https://spiffe.io/). As the [SPIFFE](https://spiffe.io/) specifications mature, we intend for Istio Auth to become a reference implementation of the same.
+
+The diagram below provides an overview of the Istio Auth service authentication architecture on Kubernetes.
+
+<figure>
+    <a href="{{home}}/img/istio_auth_overview.svg">
+    <img style="max-width: 100%;" src="{{home}}/img/istio_auth_overview.svg" alt="" title="Istio Auth Overview" />
```
Comment (geeknoid) [inline _posts/2017-05-25-0.1-auth.md]: Provide a valid alt attribute, just copy the title attribute...

File: _posts/2017-05-25-0.1-auth.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+* Rotates keys and certificates periodically. 
+
+* Revokes a specific key and certificate pair when necessary (future).
+ 
+The following diagram explains the end to end Istio Auth authentication workflow on Kubernetes:
+
+<figure>
+    <a href="{{home}}/img/istio_auth_workflow.svg">
+    <img style="max-width: 100%;" src="{{home}}/img/istio_auth_workflow.svg" alt="" title="Istio Auth Workflow" />
```
Comment (geeknoid) [inline _posts/2017-05-25-0.1-auth.md]: alt attribute.

File: _posts/2017-05-25-0.1-auth.md (hunk 2 above)
Comment (wattli) [inline _posts/2017-05-25-0.1-auth.md]: @sraygoogle , what do you think?

Comment (geeknoid) [inline _posts/2017-05-25-0.1-auth.md]: We've systematically avoided using Alpha in any of our material, this is the 0.1 release....

Comment (wattli) [inline _posts/2017-05-25-0.1-auth.md]: Used 'version 0.1 release'.

File: _posts/2017-05-25-0.1-auth.md (hunk 3 above)
Comment (wattli) [inline _posts/2017-05-25-0.1-auth.md]: done

File: _posts/2017-05-25-0.1-auth.md (hunk 4 above)
Comment (wattli) [inline _posts/2017-05-25-0.1-auth.md]: done

**Pull request merged**